PrestaShop/PrestaShop
# Import / Export features from the Stock Overview page

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Please be specific when describing the PR. <br/> Every detail helps: versions, browser/server configuration, specific module/theme, etc.
| Type?         | new feature
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/FF-30 <br>
| How to test?  | check ticket, 


Test: 
- check file import (product & combination)
- import product (see stock alert value)
- import combination (see stock alert value)
- export in stock page
- check stock alert value in stock page
- check the stock alert filter

Export tested with 11k lines in product page & stock page
